content/source/docs/enterprise/vcs/troubleshooting.html.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: enterprise2
+page_title: "Troubleshooting - VCS Providers - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-vcs-troubleshooting"
+---
+
+# Troubleshooting VCS Integration in Terraform Enterprise
+
+This page collects solutions to the most common problems our users encounter with VCS integration in Terraform Enterprise (TFE).
+
+## Bitbucket Server
+
+### Clicking "Connect organization `<X>`" with Bitbucket Server causes 500 error in Terraform Enterprise
```

Comment:
Users will now see a flash message when there an error occurs connecting the OAuth client to the VCS provider.